modules/ding_wayf_dk/modules/ding_wayf_dk_create_user/ding_wayf_dk_create_user.module

```diff
@@ -0,0 +1,397 @@
+    '#title' => t('Title'),
+    '#default_value' => isset($defaults['title']) ? $defaults['title'] : '',
+  );
+
+  $form['user']['ding_wayf_user_create_before_info']['content'] = array(
+    '#type' => 'textarea',
+    '#title' => t('Content'),
+    '#default_value' => isset($defaults['content']) ? $defaults['content'] : '',
+  );
+
+  $form['user']['ding_wayf_user_create_before_info']['link'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Link title'),
+    '#default_value' => isset($defaults['link']) ? $defaults['link'] : '',
+  );
+
+  // Accepting text.
+  $form['user']['ding_wayf_user_create_accpect_text'] = array(
+    '#type' => 'textarea',
+    '#title' => t('Accept text'),
+    '#default_value' => variable_get('ding_wayf_user_create_accpect_text', ''),
+    '#description' => t('The text that is display next to the user create accept check box.'),
+  );
+
+  // Define after create info page.
+  $form['user']['ding_wayf_user_create_after_info'] = array(
+    '#type' => 'fieldset',
+    '#tree' => TRUE,
+    '#collapsible' => TRUE,
+    '#collapsed' => TRUE,
+    '#title' => t('User creating info pager (after)'),
+    '#description' => t('Information to display on the page displayed when the user have sent a creation request.'),
+  );
+
+  $defaults = variable_get('ding_wayf_user_create_after_info', array());
+  $form['user']['ding_wayf_user_create_after_info']['title'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Title'),
+    '#default_value' => isset($defaults['title']) ? $defaults['title'] : '',
+  );
+
+  $form['user']['ding_wayf_user_create_after_info']['content'] = array(
+    '#type' => 'textarea',
+    '#title' => t('Content'),
+    '#default_value' => isset($defaults['content']) ? $defaults['content'] : '',
+  );
+}
+
+/**
+ * Implements hook_theme().
+ */
+function ding_wayf_dk_create_user_theme($existing, $type, $theme, $path) {
+  return array(
+    'ding_wayf_dk_info_page' => array(
+      'template' => 'ding-wayf-dk-info-page',
+      'variables' => array(
+        'title' => array(),
+        'content' => array(),
+        'link' => NULL,
+      ),
+      'path' => $path . '/templates',
+    ),
+    'ding_wayf_dk_create_user_link' => array(
+      'template' => 'ding-wayf-dk-create-user-link',
+      'variables' => array(
+        'link' => NULL,
+      ),
+      'path' => $path . '/templates',
+    ),
+  );
+}
+
+/**
+ * Menu callback for information page before user creation.
+ */
+function ding_wayf_dk_create_user_info_page() {
+  // Get texts from the configuration.
+  $values = variable_get('ding_wayf_user_create_before_info', array());
+
+  return array(
+    '#theme' => 'ding_wayf_dk_info_page',
+    '#title' => empty($values['title']) ? t('Not configured.') : $values['title'],
+    '#content' => empty($values['content']) ? t('Please configure WAYF configuration to get content on this page under <a href="/admin/config/people/wayf">admin/config/people/wayf</a>.') : $values['content'],
```

Comment:
Please provide a better default value. Having "Not configured" potentially exposed to end users is never a good idea.